blog/2022/01/22/o2-orion-monopoles.mdx

```diff
@@ -0,0 +1,173 @@
+---
+title: 'O2 Beacon 3 - Orion'
+description:
+  O2's latest flagship monopoles, Orion, are a mastermind in the mobile networking world, cramming two 5G carriers and tonnes of 4G bands into a
+  single streetworks cabinet and monopole. It's a work of art.
+path: 2022/01/07/o2-beacon-3-orion
+created_at: 2022-01-22T23:00Z
+---
+
+import { FactBox, MathBlock, Tooltip, SpectrumMap } from '@blog'
+
+<TableOfContents {...props} />
+
+<FactBox title="Note">
+
+Click on any of the images in this article to view them at their full-size glory.
+
+</FactBox>
+
+# A brief history
+
+O2 (<Tooltip text="MCC: 234, MNC: 10"><code>234/10</code></Tooltip>) is the largest network operator in the United Kingdom in terms of customers.
+Including its <Tooltip text="Mobile virtual network operators">MVNOs</Tooltip>, O2's customer base totals a whopping 36.2 million customers in a
+country of just 67 million, meaning they have a market share of 54%.
+([Source](https://news.o2.co.uk/press-release/o2-continues-customer-growth-and-profitability-as-it-supports-uk-through-pandemic))
+
+In the past, O2 has had a fairly troublesome history in terms of capacity. Many parts of the UK which used to be more rural have expanded over
```

Comment:
```suggestion
In the past, O2 has struggled in terms of capacity. Many parts of the UK which used to be more rural have grown over time, resulting in O2 sites which used to be more than capable of providing decent performance to customers within their zone suffering from lower performance as network load increases each year. This is extremely prevalent with their main 800 MHz LTE deployments across rural and suburban communities.
```
UK is obvious and doesn't need restated, tried to make the rest a little more positive.